packages/ui/feature-dashboard/src/lib/pages/runs/runs.component.ts

```diff
@@ -173,7 +173,8 @@ export class RunsComponent
         FlowRunStatus.INTERNAL_ERROR,
         FlowRunStatus.QUOTA_EXCEEDED,
       ],
-      issue.flowId
+      issue.flowId,
+      new Date(issue.created.split('T')[0])
```

Comment:
You are removing the timezone, I assume you are doing this to avoid the place where it set hour to 0 and end date hour to 23:59:59.

I suggest to rewrite this area.

Reply:
yes i do this to remove he time, however, it doesnt mess up the PT timezone becuase we add it in the email service separately, check attached photo
![image](https://github.com/activepieces/activepieces/assets/44944648/e06387e5-4310-44d7-a3ec-7bf8fc322be3)